gallery/visualtests/nxRadio.js

```diff
@@ -22,68 +29,38 @@ describe('NxRadio', function() {
+      const inputSelector = `${selector} input`;
 
-      const blurSelector = `${selector} input`,
-        [targetElement, otherElement, blurElement] = await Promise.all([browser.$(selector),
-          browser.$(otherRadioSelector), browser.$(blurSelector)]);
+      const [targetElement, inputElement] = await waitAndGetElements(selector, inputSelector);
 
-      await targetElement.scrollIntoView({ block: 'center' });
       await targetElement.click();
-      await browser.execute(function(el) {
-        el.blur();
-      }, blurElement);
-
-      try {
-        await browser.eyesRegionSnapshot(null, Target.region(targetElement));
-      }
-      finally {
-        // click away to reset the state
-        await otherElement.click();
-      }
+      await blurElement(inputElement);
+
+      await checkScreenshot(targetElement);
     });
 
     it(`has a thick blue border, white background, with a light blue outer border and glow
       when clicked and focused`, async function() {
       const focusSelector = `${selector} input`,
-          [targetElement, otherElement, focusElement] =
-            await Promise.all([browser.$(selector), browser.$(otherRadioSelector), browser.$(focusSelector)]);
+          [targetElement, focusElement] = await waitAndGetElements(selector, focusSelector);
 
-      await targetElement.scrollIntoView({ block: 'center' });
       await targetElement.click();
-      await targetElement.moveTo({ xOffset: -10, yOffset: -10 });
-      await browser.execute(function(el) {
-        el.focus();
-      }, focusElement);
-
-      try {
-        await browser.eyesRegionSnapshot(null, Target.region(targetElement));
-      }
-      finally {
-        // click again to reset the state
-        await otherElement.click();
-      }
+      await moveMouseAway();
+      await focusElement.focus();
+
+      await checkScreenshot(targetElement);
     });
 
     it(`has a thick blue border and white background with a light blue outer border
       when clicked, focused, and hovered`, async function() {
       const focusSelector = `${selector} input`,
-          [targetElement, otherElement, focusElement] =
-            await Promise.all([browser.$(selector), browser.$(otherRadioSelector), browser.$(focusSelector)]);
+          [targetElement, focusElement] = await waitAndGetElements(selector, focusSelector);
 
-      await targetElement.scrollIntoView({ block: 'center' });
       await targetElement.click();
-      await targetElement.moveTo();
-      await browser.execute(function(el) {
-        el.focus();
-      }, focusElement);
-
-      try {
-        await browser.eyesRegionSnapshot(null, Target.region(targetElement));
-      }
-      finally {
-        // click again to reset the state
-        await otherElement.click();
-      }
+      await focusElement.focus();
+      await targetElement.hover();
+
+      await checkScreenshot(targetElement);
```

Comment:
Same as above, do we not need to reset the state?

Reply:
> Same as above, do we not need to reset the state?

https://github.com/sonatype/sonatype-react-shared-components/pull/510#discussion_r839957163